src/redismodule.rs

```diff
@@ -145,6 +145,13 @@ impl RedisString {
         Self::from_ptr(self.inner).map_err(|_| RedisError::Str("Couldn't parse as UTF-8 string"))
     }
 
+    pub fn as_slice<'a>(&self) -> &'a [u8] {
+        let mut len: libc::size_t = 0;
+        let bytes = unsafe { raw::RedisModule_StringPtrLen.unwrap()(self.inner, &mut len) };
+
+        unsafe { slice::from_raw_parts(bytes.cast::<u8>(), len) }
```

Comment:
This duplicates the code in `to_string_lossy`; I suggest making `to_string_lossy` call this function instead.